src/braket/aws/aws_quantum_job.py

```diff
@@ -165,6 +165,11 @@ def create(
         Raises:
             ValueError: Raises ValueError if the parameters are not valid.
         """
+
+        if(type(hyperparameters) is dict):
+            keys_values = hyperparameters.items()
+            hyperparameters = {str(key): str(value) for key, value in keys_values}
```

Comment:
```suggestion
            hyperparameters = {str(key): str(value) for key, value in hyperparameters.items()}
```
nit: keys_values is not a descriptive name and doesn't need to be its own variable; `items()` is a ubiquitous accessor method for interfacing directly with a dict

Reply:
Removed.